apiserver/facades/agent/uniter/newlxdprofile.go.copy

```diff
@@ -0,0 +1,301 @@
+// Copyright 2018 Canonical Ltd.
```

Comment:
Delete file.

Reply:
damn it, thought I'd gotten rid of that.  ty